android/KMEA/app/src/main/java/com/tavultesoft/kmea/data/KeyboardController.java

```diff
@@ -171,7 +171,7 @@ public int getKeyboardIndex(String key) {
       }
     }
 
-    KMLog.LogError(TAG, "getKeyboardIndex failed for key " + key);
+    Log.w(TAG, "getKeyboardIndex failed for key " + key);
```

Comment:
I'm downgrading these from error to warnings.
I was testing with the FV oem app and saw these in the logs.

Since now we're always checking if the keyboard exists (l. 217), it's not really a fatal error if a keyboard index isn't found.